### PR TITLE
eui64: Added setting EUI64 and OUI definitions via Zephyr config.

### PR DESCRIPTION
**Pull request by kkasperczyk-no**
Setting definitions used for EUI64 source customization
and OUI vendor selection via Zephyr config options was added.

Signed-off-by: Kamil Kasperczyk <kamil.kasperczyk@nordicsemi.no>

### CODE REVIEW

**Comment by kkasperczyk-no** [comment]
CC @rlubos @MarekPorwisz

File: zephyr/CMakeLists.txt [hunk 1]
```diff
@@ -24,6 +24,12 @@ if(CONFIG_OPENTHREAD_SLAAC)
   set(OT_SLAAC ON CACHE BOOL "Enable SLAAC")
 endif()
 
+set(OT_OUI "${CONFIG_OPENTHREAD_STACK_VENDOR_OUI}" CACHE STRING "OpenThread vendor OUI address")
```
Comment (MarekPorwisz) [inline zephyr/CMakeLists.txt:27]: I was unable to find any other occurrence of OT_OUI

File: zephyr/CMakeLists.txt [hunk 2]
```diff
@@ -24,6 +24,12 @@ if(CONFIG_OPENTHREAD_SLAAC)
...
+
+if(CONFIG_OPENTHREAD_EUI64_CUSTOM_SOURCE)
+  set(OT_CUSTOM_EUI64 ON CACHE BOOL "Enable custom EUI64 source")
```
Comment (MarekPorwisz) [inline zephyr/CMakeLists.txt:30]: I was unable to find any other occurrence of OT_CUSTOM_EUI64

File: zephyr/CMakeLists.txt (hunk 1 above)
Comment (rlubos) [inline zephyr/CMakeLists.txt:27]: I don't see `OT_OUI` and `OT_CUSTOM_EUI64` options in OpenThread, are they going to be added in the near future?

**Pull request closed without merging**